manifests/vhost.pp

```diff
@@ -1853,13 +1853,13 @@
   Optional[Variant[Array[String],String]] $redirectmatch_status                       = undef,
   Optional[Variant[Array[String],String]] $redirectmatch_regexp                       = undef,
   Optional[Variant[Array[String],String]] $redirectmatch_dest                         = undef,
-  Optional[String] $headers                                                           = undef,
-  Optional[Array[String]] $request_headers                                            = undef,
-  Optional[Array[String]] $filters                                                    = undef,
-  Optional[Array] $rewrites                                                           = undef,
-  Optional[String] $rewrite_base                                                      = undef,
-  Optional[Variant[Array[String],String]] $rewrite_rule                               = undef,
```

Comment:
I don't see how it would. AFAIK this is the part that renders it:
https://github.com/puppetlabs/puppetlabs-apache/blob/1186e90f2338c33a8ae40666057a0cf8372704d6/templates/vhost/_rewrite.erb#L41-L53
I don't see a loop there. Am I missing something?

Note that the part with a loop only works for `$rewrites`:
https://github.com/puppetlabs/puppetlabs-apache/blob/1186e90f2338c33a8ae40666057a0cf8372704d6/templates/vhost/_rewrite.erb#L32-L37

Reply:
Ah, I misread the code. 
NVM